File: ArxProjects/NetOverruleTest/NetOverruleTest/PlugInApplication.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
#if ZRX
using ZwSoft.ZwCAD.Runtime;
using ZwSoft.ZwCAD.ApplicationServices;
using ZwSoft.ZwCAD.Geometry;
using ZwSoft.ZwCAD.DatabaseServices;
using ZwSoft.ZwCAD.EditorInput;
#else
using Autodesk.AutoCAD.ApplicationServices;
using Autodesk.AutoCAD.DatabaseServices;
using Autodesk.AutoCAD.EditorInput;
using Autodesk.AutoCAD.Runtime;
#endif

[assembly: ExtensionApplication(typeof(NetOverruleTest.PlugInApplication))]

namespace NetOverruleTest
{
    public class PlugInApplication : IExtensionApplication
    {
        public void Initialize()
        {
            // Add your initialize code here.
        }

        public void Terminate()
        {
            // Add your uninitialize code here.
        }
    }
}

File: ArxProjects/cadtest1/VBCADTest/VBCADTest/PluginApplication.vb
Imports System
#If ZRX Then
Imports ZwSoft.ZwCAD.Runtime
Imports ZwSoft.ZwCAD.ApplicationServices
Imports ZwSoft.ZwCAD.DatabaseServices
Imports ZwSoft.ZwCAD.Geometry
Imports ZwSoft.ZwCAD.EditorInput
#Else
Imports Autodesk.AutoCAD.Runtime
Imports Autodesk.AutoCAD.ApplicationServices
Imports Autodesk.AutoCAD.DatabaseServices
Imports Autodesk.AutoCAD.Geometry
Imports Autodesk.AutoCAD.EditorInput
#End If


<Assembly: ExtensionApplication(GetType(VBCADTest.PluginApplication))>

Namespace VBCADTest
    Public Class PluginApplication
        Implements IExtensionApplication

        Public Sub Initialize() Implements IExtensionApplication.Initialize
            'Add your initialize code here.

        End Sub

        Public Sub Terminate() Implements IExtensionApplication.Terminate
            'Add your uninitialize code here.

        End Sub

    End Class

End Namespace
